tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -985,6 +985,7 @@ def __init__(self,
                        "with static TensorRT ops. Set is_dynamic_op to True.")
 
     self._converted = False
+    self._build_called_once = False
```

Comment:
Are the changes in this file relevant? I don't see how self._build_called_once is set to true...